web/src/components/storage/SearchedDeviceMenu.tsx

```diff
@@ -261,25 +286,28 @@ const RemoveEntryOption = ({ device, onClick }: RemoveEntryOptionProps): React.R
 };
 
 const targetDevices = (
-  modelDevice: model.Drive | model.MdRaid,
-  model: Model,
+  modelDevice: configModel.Drive | configModel.MdRaid,
+  configModel: configModel.Config,
   availableDevices: storage.Device[],
 ): storage.Device[] => {
   return availableDevices.filter((availableDev) => {
     if (modelDevice.name === availableDev.name) return true;
 
-    const collection = isDrive(availableDev) ? model.drives : model.mdRaids;
+    const collection = isDrive(availableDev) ? configModel.drives : configModel.mdRaids;
     const device = collection.find((d) => d.name === availableDev.name);
     if (!device) return true;
 
-    return modelDevice.filesystem ? !device.isUsed : !device.filesystem;
+    return modelDevice.filesystem
```

Comment:
well, after change ternary operator splits over multiple lines...in such case I prefer to use simply if condition for better readability.